ls1intum/Artemis
# Add Kotlin, VHDL, Assember Programming Language Templates.

### Checklist

### Motivation and Context
Some courses that use Artemis run exercises with Kotlin, VHDL or Assembler. Currently, the setup for these exercises has to be done manually (or imported from an already existing exercise).

### Description
This PR adds templates for Kotlin, VHDL and Assembler.
The template for Kotlin is very similar to the Java template (including some non-trivial tests), whereas the VHDL and Assembler tests only contain very basic functionality (compile checks).

### Steps for Testing

1. Create a programming exercise for each of the new programming languages
2. Verify that the base build plan fails (with 0 tests passed) and that the solution build plan passes.
3. Solve the exercises (You can check https://github.com/ls1intum/Artemis/tree/develop/src/main/resources/templates in case you need some help ;-))

**Note:** The new programming languages are only supported with Bamboo, so make sure to use TS1 or TS3 when testing.

### Test Coverage

* ExerciseService.java: 85%
* programming-exercise.component.ts 95%

### Screenshots
